packages/@lwc/module-resolver/src/__tests__/module-errors.spec.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+import path from 'path';
+import { resolveModule } from '../index';
+
+describe('resolve individual module', () => {
+    test('throw when no modules are empty', () => {
+        function run() {
+            const customImporter = path.join(__dirname, 'fixtures/errors/empty/empty.js');
+            const expectedImportee = 'empty';
+
+            resolveModule(expectedImportee, customImporter);
+        }
+
+        expect(run).toThrow(
+            new Error('Unable to resolve empty: No ModuleRecords have been defined')
+        );
```

Comment:
I would recommend keeping only the minimal set of code in the expect function to avoid catching different errors. Same observation for the rest of the test cases. When matching against a standard `Error` instance no need to create a new instance for the assertion.

```suggestion
        const customImporter = path.join(__dirname, 'fixtures/errors/empty/empty.js');
        const expectedImportee = 'empty';
        
        expect(() => resolveModule(expectedImportee, customImporter)).toThrow(
            'Unable to resolve empty: No ModuleRecords have been defined'
        );
```

Reply:
Good call